tutorials/math/vector_math.rst

```diff
@@ -220,16 +230,16 @@ other object:
 
 The surface normal has a value of ``(0, -1)`` because this is a horizontal
 surface. When the ball collides, we take its remaining motion (the amount left
-over when it hits the surface) and reflect it using the normal. In Godot, the
-:ref:`Vector2 <class_Vector2>` class has a ``bounce()`` method to handle this.
-Here is a GDScript example of the diagram above using a :ref:`CharacterBody2D
+over when it hits the surface) and reflect it using the normal. In Godot, there
+is a :ref:`bounce() <class_Vector2_method_bounce>` method to handle this.
+Here is a code example of the above diagram using a :ref:`CharacterBody2D
 <class_CharacterBody2D>`:
 
-
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    # object "collision" contains information about the collision
+    # The KinematicCollision2D object returned by `move_and_collide` contains
+    # information about the collision.
```

Comment:
I wonder if its worth it to move this comment to the descriptive text, both to be able to link KinematicCollision2D and for the sake of translations. Also I agree this is an improvement over the original, but it starts to get a bit wordy for a comment compared to the length of the code.

Same for the C# version obviously. :)